src/extensions/default/HealthData/HealthDataPreview.js

```diff
@@ -45,7 +45,11 @@ define(function (require, exports, module) {
         var result = new $.Deferred();
 
         HealthDataManager.getHealthData().done(function (healthDataObject) {
-            var content = JSON.stringify(healthDataObject, null, 4);
+            var combinedHealthAnalyticsData = HealthDataManager.getAnalyticsData("pingData", "", "", ""),
+                content;
+            combinedHealthAnalyticsData = [healthDataObject, combinedHealthAnalyticsData ];
+            combinedHealthAnalyticsData.join("\n\n");
+            content = JSON.stringify(combinedHealthAnalyticsData, null, 4);
```

Comment:
By joining the elements in the previous statement we could have produced a string already. Why are we serializing it again?

Reply:
This is addressed in current commit